### PR TITLE
vv 00

### PR DESCRIPTION
**Pull request by V-tyna**
Blue box  block implemented.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<meta charset="UTF-8" />
+		<title>PR test</title>
+		<link rel="stylesheet" href="./styles.css" />
```
Comment (AntipMen) [inline index.html:5]: I think this field should be before title

File: index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+console.log('Works');
```
Comment (AntipMen) [inline index.js]: I think is redundant

**Review by AntipMen — CHANGES_REQUESTED** [review]
Please, update your code

File: index.html (hunk 1 above)
Comment (V-tyna) [reply index.html:5]: Done

**Review by AntipMen — APPROVED** [review]

**Pull request merged**